pandas/tests/io/test_xml.py

```diff
@@ -237,6 +236,28 @@ def test_file_buffered_reader_no_xml_declaration(datapath, parser, mode):
     tm.assert_frame_equal(df_str, df_expected)
 
 
+@td.skip_if_no("lxml")
+def test_closed_file_lxml(datapath):
```

Comment:
you can probably parametrize these two tests.

In general, I'm not sure whether it is necessary to enforce generic error messages for "obviously" wrong inputs (None/closed files handles). @jreback 

One test to add (or extending an existing test) is to make sure that a user-provided file handle is not closed by read/to_xml.

Reply:
Understood. I can remove those wrong input tests. I tried simulating how users may behave (having answered many StackOverflow pandas answers from newbies!). Will parametrize and add file handle close tests.